src/test/java/org/squiddev/cobalt/ScriptHelper.java

```diff
@@ -122,6 +126,14 @@ public void runComparisonTest(String testName) throws Exception {
 			ps.flush();
 			String actualOutput = new String(output.toByteArray());
 			String expectedOutput = getExpectedOutput(testName);
+
+			Path dir = Paths.get("out");
+			Files.createDirectories(dir);
+			try ( BufferedWriter out = Files.newBufferedWriter(
```

Comment:
I don't know how you're running tests, but I know IntelliJ gives you a diff view for `assertEquals` failures.

It might be worth writing this file into a subdirectory, just so it's not filling the root dir (though I guess that's already an issue with some of the other tests).

Reply:
Did I check that in?  This was a quick and dirty so I could get a full file diff of the test output.  IntelliJ seemed to balk at that large of a difference.  I'll take it out right away.